src/gatsby-theme-docz/wrapper.js

```diff
@@ -2,9 +2,13 @@ import React from "react";
 import PropTypes from "prop-types";
 import { Helmet } from "react-helmet-async";
 
-// Note: We are using react-helmet-async here instead of react-helmet
-// https://github.com/nfl/react-helmet/issues/426
-// https://www.docz.site/docs/add-favicon-and-metadata
+/**
+ * Note: We're referencing `react-helmet-async`, and not `react-helmet`. This
+ * is because of this (https://github.com/nfl/react-helmet/issues/426) issue.
+ * `react-helmet-async` is an API-compatible fork.
```

Comment:
I guess I was hoping that we could put a short description of what that issue is (in our context), instead of making them follow the link.

Reply:
The issue is from Docz context. So, I moved their explanation up to this level. This isn't something we've added in ourselves if that helps?

Reply:
Changed it a bit perhaps that is better?

Reply:
Looks good, thanks!